networkmanager.cpp

```diff
@@ -143,7 +140,7 @@ void NetworkManager::on_srv_lookup()
       timer.start();
       do
       {
-        ao_app->processEvents();
+        force_process_events();
```

Comment:
```suggestion
        emit force_process_events();
```
(Should this be a signal though?)

Reply:
What else do you suggest?

Reply:
You should make it a regular function - signals are for events.

Reply:
Right, but the `processEvents()` is still in the `ao_app`, since that's the `QApplication`.
So I do have to signal to it, no?

Reply:
Oh, I see.